zer0-os/zOS
# feat(redesign-message-input): update input positioning/styles and add placeholder to mentions component

### What does this do?
- updates message input positioning/styles and adds placeholder to mentions component

### Why are we making this change?
- as per redesign requests

### How do I test this?
- run tests as usual
- run UI > Check message input aligns with design (as per screenshots below)

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

Before


<img width="1566" alt="Screenshot 2024-09-11 at 12 23 05" src="https://github.com/user-attachments/assets/17b0d51c-9e8f-4aff-8bf7-48705db706a6">

After
<img width="1566" alt="Screenshot 2024-09-11 at 12 22 49" src="https://github.com/user-attachments/assets/ebf6a895-4197-498d-acce-5da73e15aa31">
